hyp3_gamma/__main__.py

```diff
@@ -226,5 +226,75 @@ def water_map():
             upload_file_to_s3(product_file, args.bucket, args.bucket_prefix)
 
 
+def flood_map():
```

Comment:
This looks pretty good! I think we only need one "water" job, however, so you could combine this with the water map job. Essentially, just adding the second conda execute and extra args up to the water map job.

I think we had talked with Franz about it being an extra layer (you can confirm with him), so you'd also need an `--include-flood-depth` true/false option that would toggle running the second execute

Reply:
Great feedback! I deleted `flood_map()` and added the execution after `if args.include_flood_map:` in `water_map()`.